blocks/api/raw-handling/utils.js

```diff
@@ -63,8 +73,18 @@ export function isAttributeWhitelisted( tag, attribute ) {
 	);
 }
 
-export function isInline( node ) {
-	return !! inlineWhitelist[ node.nodeName.toLowerCase() ];
+function nodeIsInlineOnSpecificTag( nodeName, tagName ) {
+	if ( ! tagName || ! nodeName ) {
+		return false;
+	}
+	return includes(
+		find( inlineWhitelistTagGroups, group => includes( group, tagName.toUpperCase() ) ),
+		nodeName,
+	);
+}
+
+export function isInline( node, tagName ) {
+	return !! inlineWhitelist[ node.nodeName.toLowerCase() ] || nodeIsInlineOnSpecificTag( node.nodeName, tagName );
```

Comment:
I suggest adding `const nodeName = node.nodeName.toLowerCase()` to simplify the inner function calls, as well as standardize on always using lowercase tag names in that module.

Reply:
The name `nodeIsInlineOnSpecificTag` is longer but not necessarily clearer, because the entire problem requires explaining with more than a few words. :) Suggestions: `isInlineForTag`, `isInlineGivenTag`, `isInlineForBaseTag`, complemented with a jsdoc for that function.